src/main/java/com/linkedin/kafka/clients/producer/LiKafkaProducerImpl.java

```diff
@@ -196,61 +198,115 @@ private LiKafkaProducerImpl(LiKafkaProducerConfig configs,
       V value = producerRecord.value();
       Object auditToken = _auditor.auditToken(key, value);
```

Comment:
Let's discuss changes to the audit API with Radai when he gets back.  This is a change I would push out to some future PR.